galleries/examples/color/color_sequences.py

```diff
@@ -38,8 +38,8 @@ def plot_color_sequences(names, ax):
 
 built_in_color_sequences = [
     'tab10', 'tab20', 'tab20b', 'tab20c', 'Pastel1', 'Pastel2', 'Paired',
-    'Accent', 'Dark2', 'Set1', 'Set2', 'Set3', 'petroff6', 'petroff8',
-    'petroff10']
+    'Accent', 'Accessible', 'Dark2', 'Set1', 'Set2', 'Set3', 'petroff6',
```

Comment:
```suggestion
    'Accent', 'okabe-ito', 'Dark2', 'Set1', 'Set2', 'Set3', 'petroff6',
```

Many of these sequences are accessible on different dimensions. Users should choose the pallet that fulfills their constraints, rather than MPL declaring any one sequence the "accessible" one.

Reply:
License is MIT.
Okay for the name change, Okabe-Ito (capitalized) sounds good, unless there are better suggestions? Also it could be good to have a hint at the accessibility level next to the name by appending a descriptive suffix like Okabe-Ito_cvd or Okabe-Ito_c. This should be then done for all accessible designed sequences.